src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -129,8 +128,8 @@ public InitCodeView generateSmokeTestInitCode(
             InitTreeParserContext.newBuilder()
                 .table(table)
                 .rootObjectType(testConfig.getMethod().getInputType())
-                .initValueConfigMap(createInitValueMap(context))
-                .dottedPathStrings(testConfig.getInitFields())
+                .initValueConfigMap(createCollectionMap(context))
+                .initFieldConfigStrings(testConfig.getInitFieldSpecs())
```

Comment:
Why rename with Specs? It seems like we have removed the Specs naming convention from everywhere else